src/test/interface/stansummary_test.cpp

```diff
@@ -198,59 +202,6 @@ TEST(CommandStansummary, percentiles) {
   EXPECT_THROW(percentiles_to_probs(pcts), std::invalid_argument);
 }
 
-TEST(CommandStansummary, param_tests) {
```

Comment:
Did this test get replaced somewhere I'm not seeing?

Reply:
this test is redundant with all of the tests which check console and CSV outputs.